.gitignore

```diff
@@ -28,3 +28,8 @@ docs/html
 
 # Do not track results
 regression_tests/results
+.gitignore
+
+
```

Comment:
I'm not sure these changes to gitignore are useful for other people. I don't see `.gitignore` in my environment, and I think a git setting will cause it to be ignored automatically. Also, I don't copy `astyle` to my libROM directory. Maybe these changes should just be kept in your local version.

Reply:
Yeah sorry, this should not be here...